src/backend/joanie/core/utils/contract.py

```diff
@@ -149,3 +151,19 @@ def generate_zip_archive(pdf_bytes_list: list, user_uuid: str, zip_uuid=None) ->
     storage.save(name=zip_archive_name, content=zip_buffer)
 
     return zip_archive_name
+
+
+def organization_owners_presence_for_signing(order_id=None) -> bool:
```

Comment:
Maybe something simpler like
```suggestion
def organization_has_owner(order_id=None) -> bool:
```
as signing is not directly involved in this function.